src/wazuh_db/wdb.h

```diff
@@ -715,10 +715,10 @@ int wdb_package_update(wdb_t * wdb, const char * scan_id);
 int wdb_package_delete(wdb_t * wdb, const char * scan_id);
 
 // Insert process info tuple. Return 0 on success or -1 on error.
-int wdb_process_insert(wdb_t * wdb, const char * scan_id, const char * scan_time, int pid, const char * name, const char * state, int ppid, int utime, int stime, const char * cmd, const char * argvs, const char * euser, const char * ruser, const char * suser, const char * egroup, const char * rgroup, const char * sgroup, const char * fgroup, int priority, int nice, int size, int vm_size, int resident, int share, int start_time, int pgrp, int session, int nlwp, int tgid, int tty, int processor, const char * checksum, const bool replace);
+int wdb_process_insert(wdb_t * wdb, const char * scan_id, const char * scan_time, int pid, const char * name, const char * state, int ppid, int utime, int stime, const char * cmd, const char * argvs, const char * euser, const char * ruser, const char * suser, const char * egroup, const char * rgroup, const char * sgroup, const char * fgroup, int priority, int nice, int size, int vm_size, int resident, int share, long long start_time, int pgrp, int session, int nlwp, int tgid, int tty, int processor, const char * checksum, const bool replace);
```

Comment:
Hi @jnasselle 

Is there a possibility that the `start_time` field grows large enough to produce an erroneous read if it is signed? In fact, the conversion used in the linuxInfoHelper.h file was from string to unsigned long long.

Best regards
Hanes

Reply:
Hi @HanesSciarrone ,

You are right, but sqlite3 db does not support storing unsigned integers. That's why the helper retrieve `uint64_t` but is statically casted to `int64_t` when this data is returned by sysinfo (this data will be sync to local agent db using signed integer).

Regards,
Nico